ReactiveObjC/RACTuple.h

```diff
@@ -49,7 +49,7 @@
 /// +tupleNil.
 @interface RACTupleNil : NSObject <NSCopying, NSCoding>
 /// A singleton instance.
-+ (RACTupleNil *)tupleNil;
++ (instancetype)tupleNil;
```

Comment:
This is incorrect. It returns a specific class—not an instance of the current class.